### PR TITLE
kill dead wood

### PR DESCRIPTION
**Pull request by Rick-Anderson**
I was hoping to make this generic so we could use it, but I don't think that's possible. It was too painful to copy so I did some cleanup. We should consider making this generic. If we do, I can help clean it up and maintain it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,13 +55,13 @@ Also, remove the "wrench" icon (🔧) from the TOC and the file heading, if appl
 
 If your PR is addressing an existing issue, add the `Fixes #Issue_Number` keyword to the commit message or PR description, so the issue can be automatically closed when the PR is merged. For more information, see [Closing issues via commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/).
 
-The .NET team will review your PR and let you know if the change looks good or if there are any other updates/changes necessary in order to approve it.
+The .NET team will review your PR and let you know if there are any other updates/changes necessary in order to approve it.
 
-**Step 6:** Make any necessary updates to your branch as discussed with the team. 
+**Step 6:** Terspond to the PR feedback.
```
Comment (mairaw) [inline CONTRIBUTING.md]: Terspond -> Respond

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -58,13 +55,13 @@ Also, remove the "wrench" icon (🔧) from the TOC and the file heading, if appl
...
 
-The maintainers will merge your PR into the master branch once feedback has been applied and your change looks good. 
+The maintainers will merge your PR into the master branch once feedback has been applied and your PR is approved.
 
-On a certain cadence, we push all commits from master branch into the live branch and then you'll be able to see your contribution live at https://docs.microsoft.com/dotnet/. 
+We typically push all commits from master branch into the live branch. You'll be able to see your contribution live at https://docs.microsoft.com/dotnet/ within an hour of the contribution being merged.
```
Comment (mairaw) [inline CONTRIBUTING.md]: I wouldn't add any expectations around when changes show up live and it's not that we typically push, we do have to push always.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -75,19 +72,18 @@ Below is a short list of guiding rules that you should keep in mind when you are
...
+- **DO** blog, tweet and use other social media to annouce your contribution.
 
 > Note: you might notice that some of the topics are not currently following all the guidelines specified here and on the [style guide](./styleguide/template.md) as well. We're working towards achieving consistency throughout the site. Check the list of [open issues](https://github.com/dotnet/core-docs/issues?q=is%3Aissue+is%3Aopen+label%3Aguidelines-adherence) we're currently tracking for that specific goal. 
 
 ## Building the docs
 
 The documentation is written in [GitHub Flavored Markdown](https://help.github.com/categories/writing-on-github/) and built using [DocFX](http://dotnet.github.io/docfx/) and other internal publishing/building tools. It is hosted at [docs.microsoft.com](https://docs.microsoft.com/dotnet). 
 
-To build the docs locally, you need to install [DocFX](https://dotnet.github.io/docfx/); latest versions are the best.
+Install [DocFX](https://dotnet.github.io/docfx/) to build the docs.
```
Comment (mairaw) [inline CONTRIBUTING.md]: to build the docs locally

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good to me but I have a few comments that I left for you to address. Thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (Rick-Anderson) [inline CONTRIBUTING.md]: got it. But the original doesn't make much sense and seems to imply quick viewing .

File: CONTRIBUTING.md (hunk 3 above)
Comment (Rick-Anderson) [inline CONTRIBUTING.md]: Why do you need locally? That's just confusing to me. How else would you build it? I guess it doesn't hurt to leave in locally. I'll add it.

Comment (mairaw) [inline CONTRIBUTING.md]: there are two types of builds: local and automatically triggered OPS builds. not sure how locally can be confusing.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -58,14 +55,13 @@ Also, remove the "wrench" icon (🔧) from the TOC and the file heading, if appl
 
 If your PR is addressing an existing issue, add the `Fixes #Issue_Number` keyword to the commit message or PR description, so the issue can be automatically closed when the PR is merged. For more information, see [Closing issues via commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/).
 
-The .NET team will review your PR and let you know if the change looks good or if there are any other updates/changes necessary in order to approve it.
-
-**Step 6:** Make any necessary updates to your branch as discussed with the team. 
+The .NET team will review your PR and let you know if there are any other updates/changes necessary in order to approve it.
 
-The maintainers will merge your PR into the master branch once feedback has been applied and your change looks good. 
+**Step 6:** Rerspond to the PR feedback.
```
Comment (mairaw) [inline CONTRIBUTING.md]: still doesn't say Respond 😄

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -58,14 +55,13 @@ Also, remove the "wrench" icon (🔧) from the TOC and the file heading, if appl
...
 
-On a certain cadence, we push all commits from master branch into the live branch and then you'll be able to see your contribution live at https://docs.microsoft.com/dotnet/. 
+The maintainers will merge your PR into the master branch once feedback has been applied and your PR is approved.
 
+We periodically push all commits from master branch into the live branch. You'll be able to see your contribution at https://docs.microsoft.com/dotnet/, typically within a few hours of the merge.
```
Comment (mairaw) [inline CONTRIBUTING.md]: which merge are you referring to in the last sentence? the merge to master or the merge to live? we don't always merge to live the same day, so I'm just trying to not create false expectations here.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -75,19 +71,18 @@ Below is a short list of guiding rules that you should keep in mind when you are
 - **DO** use the [template](./styleguide/template.md) file as the starting point of your work.
 - **DO** create a separate branch on your fork before working on the articles.
 - **DO** follow the [GitHub Flow workflow](https://guides.github.com/introduction/flow/). 
-- **DO** blog and tweet (or whatever) about your contributions, frequently!
+- **DO** blog, tweet and use other social media to annouce your contribution.
```
Comment (mairaw) [inline CONTRIBUTING.md]: annouce -> announce

File: CONTRIBUTING.md (hunk 5 above)
Comment (Rick-Anderson) [inline CONTRIBUTING.md]: I'll add merge to live.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C28%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610280041176494-1184%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C28%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610280102252882-1184%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C28%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610280058373571-1184%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C28%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610280059381615-1184%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C28%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610280118160019-1184%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C28%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610280225244961-1184%5Cworkflow_report.html).

**Comment by saldana** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C28%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201610281928497909-1184%5Cworkflow_report.html).

**Comment by mairaw** [comment]
I'm going through our old PRs. I'll take a look at the current contrib guide for ASP.NET, these changes and merge them into my existing PR for the contributing guide on PR #1081. Thanks @Rick-Anderson!

**Pull request closed without merging**